packages/topkit-analytics-charts/src/CategoriesChartTooltip/CategoriesChartTooltip.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { FC } from 'react'
+import { Paper, Container, Typography } from '@toptal/picasso'
+
+import { Bar, Tooltips } from '../CategoriesChart/types'
+import { getDisplayTexts } from './utils'
+
+export type Props = {
+  active: boolean
+  payload: { payload: { name: string; team: number; user: number } }[]
+  tooltips: Tooltips
+  originalData: Bar[]
+}
+
+const CategoriesChartTooltip: FC<Props> = ({
+  active,
+  payload,
+  tooltips,
+  originalData
+}) => {
+  if (active && payload && payload.length > 0) {
+    const currentData = originalData.find(
+      ({ id: dataId }) => dataId === payload[0].payload.name
+    )
+
+    if (!currentData) {
+      return null
+    }
+
+    const { teamTexts, userTexts } = getDisplayTexts({
+      currentData,
+      tooltips,
+      currentPayload: payload[0].payload
+    })
+
+    return (
+      <Paper data-testid='tooltip'>
```

Comment:
It's not a tooltip but content.

```suggestion
      <Paper data-testid='tooltip-content'>
```